content/insiders/_index.md

```diff
@@ -0,0 +1,26 @@
+---
+title: "LocalStack Insiders"
+foo: "foobar"
+description: "Join the LocalStack Insiders program to help us shape the future of LocalStack! Test drive new features and engage in conversations with us so we can learn from your specific use cases."
+date: 2023-07-11T08:38:36Z
+lastmod: 2023-07-11T08:38:03Z
+draft: false
+images: []
+---
+
+### Fellow Developers!
+
+We are launching the LocalStack Insiders program, and for the initial installment, we are seeking users of all levels of experience who are utilizing LocalStack for non-commercial purposes.
+
+We are currently working on making LocalStack more accessible and to bring it to a wider audience. As a LocalStack Insider you will:
+
+- Get early access to our improved developer onboarding experience
+- Get access to our Insiders Slack channel
+- Join our commnity office hours and interact with the LocalStack team
+- Participate in our feedback program & receive exciting swag
```

Comment:
`receive some exciting swags`

Reply:
'swag' is uncountable just like 'water'. i.e. there is no plural form. 'today I drank a lot of water' --> 'today I received a lot of swag'